### PR TITLE
Update from 5 Dec 2022

### LINKED ISSUE #26: Add DOD URL
http://www.cultureready.org/ from the Defense Language and National Security Education Office

### LINKED ISSUE #37: The U.S. Army Band - DoD
Please add usarmyband.com to the list. Our site has been around for many years, but for some reason it doesn’t appear to be on this list. The U.S. Army Band is a military unit, part of the U.S. Army Military District of Washington.

"The United States Army Band "Pershing's Own" provides musical support for the leadership of the United States, to include all branches of government, and to a wide spectrum of national and international events in order to connect the Army to the American people."

### LINKED ISSUE #39: Update OK DHS links
- remove okdhs.org, redirects to https://oklahoma.gov/okdhs.html
- add okdhslive.org
- add okhotline.org

Resolves ZD 22746

### LINKED ISSUE #55: Submission of VA non-.gov Domains
INC3693270

In accordance with M-22-09, Moving the U.S. Government Toward Zero Trust Cybersecurity Principles, Section III, Applications and Workloads, Action 5.  Attached is the list non-.gov hostnames used by VA, excluding items already contained on the Government Domains Outside .Gov and .Mil listing.

[non-gov-submission 25 Mar 2022 (Veterans Affairs).csv](https://github.com/GSA/govt-urls/files/8362862/non-gov-submission.25.Mar.2022.Veterans.Affairs.csv)

Derrick Curtis
Sr. Technical Advisor
Software Product Management (SPM)

### PR DESCRIPTION
**Pull request by h-m-f-t**
Resolves #26, closes #37, fixes #39, resolves #55

### CODE REVIEW

File: 1_govt_urls_full.csv [hunk 1]
```diff
@@ -9216,4 +9216,12 @@ smithsonianearthtv.com,Smithsonian Institution,,,Federal,Executive,,,,
...
-otianywhere.net,United States Agency for International Development ,Conflict Prevention and Stabilization (CPS)/Office of Transition Initiative Anywhere,Website - password protected,Federal,Executive,,Office of Transition Initiative Anywhere is using this website for their user's authentication purposes. This website and the system associated with it (OTIAnyWhere) is planned to be decommissioned in August 2022  ,,
\ No newline at end of file
+otianywhere.net,United States Agency for International Development ,Conflict Prevention and Stabilization (CPS)/Office of Transition Initiative Anywhere,Website - password protected,Federal,Executive,,Office of Transition Initiative Anywhere is using this website for their user's authentication purposes. This website and the system associated with it (OTIAnyWhere) is planned to be decommissioned in August 2022  ,,
+shopvcs.com,Department of Veterans Affairs,,Website,Federal,Executive,,Redirecrts to https://shopvcs.va.gov/,,
```
Comment (dawnpm) [inline 1_govt_urls_full.csv:9220]: typo here

File: 1_govt_urls_full.csv [hunk 2]
```diff
@@ -9216,4 +9216,12 @@ smithsonianearthtv.com,Smithsonian Institution,,,Federal,Executive,,,,
...
+treatmentworksforvets.org,Department of Veterans Affairs,,Website,Federal,Executive,,"5 Dec 2022: no web or NS, but the domain still appears registered ",,
+vaitcampus.com,Department of Veterans Affairs,IT Workforce Development,Website,Federal,Executive,,"Prompts for PIV and includes a link to the VA VDP. ""In the process of converting to a VA.GOV address.  Anticipate completion prior to end of PoP (July 22nd).""",,
+vavmc.com,Department of Veterans Affairs,Employee Education System,Website,Federal,Executive,,"Banner says ""Powered by the Veterans Health Administration, Employee Education System""",,
+okdhslive.org,,OK Human Services Department,,State,Executive,Oklahoma,,,
```
Comment (dawnpm) [inline 1_govt_urls_full.csv:9224]: Does branch apply at the State level?

**Review by dawnpm — CHANGES_REQUESTED** [review]
a typo and a question, thanks.

**Comment by h-m-f-t** [comment]
Bumping this again 👋

**Review by dawnpm — APPROVED** [review]

**Pull request merged**